sigs/community-outreach.md

```diff
@@ -14,7 +14,6 @@ evaluators, and adopters. This SIG aims to accomplish this mission as follows:
 
 * Email the [Konflux mailing list](https://groups.google.com/g/konflux), optionally adding
   `[sig-community-outreach]` to the subject header.
-* We will host a recurring open SIG meeting at a time TBD.
```

Comment:
Doh! I'm sorry I forgot to adjust the repo itself when I made the meeting/calendar change :-/ . 
@ralphbean WDYT about moving the whole content of this page to the main README, dropping any SIG mentions along the way and just treat this as - yeah, all community efforts happen as part of the main meeting or at least that's how we'd want it?

Reply:
For the outreach SIG that may make sense, but I'd like the Fedora SIG to continue to exist in some form as an entity. I'm hopeful to draw some people around the activities of running and maintaining the Fedora cluster.

Reply:
That's fair, let's keep the Fedora SIG doc as is for now.